app/controllers/candidate_interface/application_form_controller.rb

```diff
@@ -73,6 +73,13 @@ def review_submitted
       @application_form = current_application
     end
 
+    def review_previous_application
+      @application_form = ApplicationForm.find(current_application.previous_application_form_id)
```

Comment:
This needs to be scoped to the current user.

Reply:
isn't current application scoped to the user?